eng/versioning/version_client.txt

```diff
@@ -178,6 +178,7 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-email;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-developer-loadtesting;1.0.0-beta.2;1.0.0
 com.azure:azure-identity-extensions;1.1.1;1.2.0-beta.2
+com.azure:azure-analytics-purview-sharing;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
nit: keep these alphabetised but I see it the trend was broken a few lines back. 😄 never mind.